magma/backend/mlir/hardware_module.py

```diff
@@ -895,8 +898,8 @@ def visit_instance_wrapper(self, module: ModuleWrapper) -> bool:
                 return True
             return self.visit_array_slice(module)
         if inst_wrapper.name.startswith("magma_array_create_op"):
-            self._make_concat(list(reversed(module.operands)),
-                              module.results[0])
+            self.make_concat(list(reversed(module.operands)),
+                             module.results[0])
```

Comment:
while we're here, nit:
```suggestion
            self.make_concat(
                list(reversed(module.operands)),
                module.results[0]
            )
```